openssl-dynamic/src/main/java/io/netty/internal/tcnative/SSLContext.java

```diff
@@ -255,67 +203,187 @@ public static boolean setCertificateBio(
      * @param keyBio Private Key BIO to use if not in cert.
      * @param password Certificate password. If null and certificate
      *                 is encrypted, password prompt will be displayed.
+     * @return {@code true} if successful
+     * @throws Exception if an error happened
      */
-    public static native boolean setCertificateBio(
-            long ctx, long certBio, long keyBio, String password) throws Exception;
+    public static native boolean setCertificateBio(long ctx, long certBio, long keyBio, String password) throws Exception;
 
     /**
      * Set the size of the internal session cache.
      * http://www.openssl.org/docs/ssl/SSL_CTX_sess_set_cache_size.html
+     * @param ctx Server or Client context to use.
+     * @param size the size of the cache
+     * @return the previous set value
      */
     public static native long setSessionCacheSize(long ctx, long size);
 
     /**
      * Get the size of the internal session cache.
      * http://www.openssl.org/docs/ssl/SSL_CTX_sess_get_cache_size.html
+     * @param ctx Server or Client context to use.
+     * @return the current value
      */
     public static native long getSessionCacheSize(long ctx);
 
     /**
      * Set the timeout for the internal session cache in seconds.
      * http://www.openssl.org/docs/ssl/SSL_CTX_set_timeout.html
+     * @param ctx Server or Client context to use.
+     * @param timeoutSeconds the timeout of the cache
+     * @return the previous set value
      */
     public static native long setSessionCacheTimeout(long ctx, long timeoutSeconds);
 
     /**
      * Get the timeout for the internal session cache in seconds.
      * http://www.openssl.org/docs/ssl/SSL_CTX_set_timeout.html
+     * @param ctx Server or Client context to use
+     * @return the current value
      */
     public static native long getSessionCacheTimeout(long ctx);
 
     /**
      * Set the mode of the internal session cache and return the previous used mode.
+     * @param ctx Server or Client context to use
+     * @param mode the mode of the cache
+     * @return the previous set value
      */
     public static native long setSessionCacheMode(long ctx, long mode);
 
     /**
      * Get the mode of the current used internal session cache.
+     *
+     * @param ctx Server or Client context to use
+     * @return the current mode
      */
     public static native long getSessionCacheMode(long ctx);
 
     /**
      * Session resumption statistics methods.
      * http://www.openssl.org/docs/ssl/SSL_CTX_sess_number.html
+     * @param ctx Server or Client context to use
+     * @return the current number
      */
     public static native long sessionAccept(long ctx);
+
+    /**
+     * Session resumption statistics methods.
+     * http://www.openssl.org/docs/ssl/SSL_CTX_sess_number.html
```

Comment:
Wrong link (404) here and in similar. Need change `/ssl/` to `/manmaster/man3/`: https://www.openssl.org/docs/manmaster/man3/SSL_CTX_sess_number.html

And are more usable when links looks as links )
`<a href="http://www.openssl.org/docs/ssl/SSL_CTX_sess_number.html">SSL_CTX_sess_number</a>`